grequests.py

```diff
@@ -73,6 +75,8 @@ def send(self, **kwargs):
         except Exception as e:
             self.exception = e
             self.traceback = traceback.format_exc()
+            if self.exception_handler:
+                self.exception_handler(self, self.exception)
```

Comment:
To be consistent with the `exception_handler` parameter of [`map`](https://github.com/kennethreitz/grequests/blob/4a15e9f5a19de42dd7831ac8372a3bbe51942c32/grequests.py#L125) and [`imap`](https://github.com/kennethreitz/grequests/blob/4a15e9f5a19de42dd7831ac8372a3bbe51942c32/grequests.py#L151)